install/update.php

```diff
@@ -836,7 +837,7 @@ function updateDbUpTo031() {
    }
 
    // Update version number and default langage and new version_founded ---- LEAVE AT THE END
-   Config::setConfigurationValues('core', array('version'             => GLPI_VERSION,
+   Config::setConfigurationValues('core', array('version'             => GLPI_SCHEMA_VERSION,
```

Comment:
No, since version checking will rely on `GLPI_SCHEMA_VERSION` (https://github.com/glpi-project/glpi/pull/1718/files#diff-412dccbbe00f02a27160b8898094e4d7R217)